.gitattributes

```diff
@@ -32,4 +32,5 @@ tooling.yml export-ignore
 .editorconfig export-ignore
 .gitattributes export-ignore
 .gitignore export-ignore
-ci/
+.github export-ignore
+ci/ export-ignore
```

Comment:
This folder should be gone now, we can remove this.

Reply:
Done.